examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -171,7 +171,7 @@ class WebGPUNodeBuilder extends NodeBuilder {
 
 			if ( alphaTest !== null ) {
 
-				alphaTest = this.addFlow( 'fragment', new VarNode( alphaTest, 'AlphaTest', 'float' ) );
+				this.addFlow( 'fragment', new VarNode( alphaTest, 'AlphaTest', 'float' ) );
```

Comment:
It seems the assignment to `alphaTest` is not necessary since the variable is not further used in the method.

Reply:
This is because if someone use `alphaTest` in rest of the flow is necessary to use the` VarNode`, would be to avoid future mistakes, I tried to create a pattern for this, but it can be done thus too:

```js
alphaTest = new VarNode( alphaTest, 'AlphaTest', 'float' );
this.addFlow( 'fragment', alphaTest );
```

Reply:
Okay, when the variable `alphaTest` is actually required, this pattern looks clear to me.

It's just that lgtm complain if we assign to `alphaTest` but not use in further the code.

Reply:
@Mugen87 I am neutral about it, for me okay remove the assignment for now, I would just like to let you know.